### PR TITLE
Ecuador's geopolitical information / Información geopolítica de Ecuador

### PR DESCRIPTION
**Pull request by coodec**
Adds the module to define the state, canton and parish on partners.

### CODE REVIEW

**Comment by coodec** [comment]
@max3903 could you please review this code.

**Comment by max3903** [comment]
:+1:

File: l10n_ec_ote/README.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Ecuador's Geopolitical Information
+===================================
```
Comment (max3903) [inline l10n_ec_ote/README.rst:7]: You have one extra `=`

**Comment by max3903** [comment]
@coodec You also have an rst syntax error:
https://travis-ci.org/OCA/l10n-ecuador/jobs/93201871#L263

**Pull request closed without merging**